packaging/nnstreamer.spec

```diff
@@ -388,6 +391,8 @@ NNStreamer's tensor_fliter subplugin of caffe2
 %package devel
 Summary:	Development package for custom tensor operator developers (tensor_filter/custom)
 Requires:	nnstreamer-core = %{version}-%{release}
+Requires:	nnstreamer-configuration = %{version}-%{release}
+Recommends: nnstreamer-test-devel = %{version}-%{release}
```

Comment:
Why do you recommend a test package with devel? test-devel is for gbs-unit-test only.

Reply:
Right, we don't even need to only `require: nnstreamer-core` so reverted this back to `Requires: nnstreamer`